misago/users/management/commands/removeips.py

```diff
@@ -1,14 +1,12 @@
-
 from django.contrib.auth import get_user_model
 from django.core.management import BaseCommand
+from misago.core.utils import ANONYMOUS_IP
+from misago.users.signals import anonymize_old_ips
 
-UserModel = get_user_model()
-
+MISAGO_IP_STORE_TIME = 33300
```

Comment:
This setting should be set in `misago/conf/defaults.py` with `MISAGO_IP_STORE_TIME = None` as well as in `misago/project_template/project_name/settings.py` as `MISAGO_IP_STORE_TIME = 50`. We'll assume this is number of days. Doesn't make sense to keep IP's for less IMHO.

Settings defined in `settings.py` can be accessed through the `misago.conf.settings.MISAGO_IP_STORE_TIME`.